From e25e8484b8643e36c65d926fb3e8c4924930b7c1 Mon Sep 17 00:00:00 2001
From: Viktor Rosendahl <viktor.rosendahl@gmail.com>
Date: Sun, 23 Feb 2020 12:22:23 +0100
Subject: [PATCH] taskgraph.cpp: Fix include path

commit e25e8484b8643e36c65d926fb3e8c4924930b7c1 upstream

We should not include qcustomplot/qcustomplot.h directly anymore
because we might be using the system qcustomplot.

This was an oversgith when I implemented the USE_SYSTEM_QCUSTOMPLOT
compile time option.

Signed-off-by: Viktor Rosendahl <viktor.rosendahl@gmail.com>
---
 ui/taskgraph.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/taskgraph.cpp b/ui/taskgraph.cpp
index e41c85a..7c66c7a 100644
--- a/ui/taskgraph.cpp
+++ b/ui/taskgraph.cpp
@@ -50,7 +50,7 @@
  *     EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include "qcustomplot/qcustomplot.h"
+#include "ui/qcustomplot.h"
 #include "ui/taskgraph.h"
 #include "analyzer/task.h"
 
-- 
2.20.1

